py/objint_longlong.c

```diff
@@ -40,14 +40,6 @@
 
 #if MICROPY_LONGINT_IMPL == MICROPY_LONGINT_IMPL_LONGLONG
 
-// Python3 no longer has "l" suffix for long ints. We allow to use it
```

Comment:
> this shows why unrelated changes should be included in one commit

Yes sorry, at the time of writing it looked all very related and non-confusing to me as I just spent some time familiarizing with all aspects of it. But in hindsight it is not all that obvious.

Commit message is fixed and commits rebased.

> That's probably because this "anti-pattern" is awfully practical

Indeed, in absence of fully configurable logging with named loggers and filtering etc, it is imo not really an anti-pattern anymore and it works very well. The opposite can also be an anti-pattern: I used some library once (don't remember which one) where enabling one single definition globally resulted in spitting out thousands of log lines. Finding the ones of interest isn't exactly a pleasure then.